py/foundation/entity.py

```diff
@@ -153,16 +157,16 @@ def children(self) -> Iterable[Entity]:
 
 
 @dataclass(frozen=True)
-class Currency(Entity):
+class DollarAmount(Entity):
```

Comment:
Does this mean we won't support non-dollar-based currencies? I think most NER engines call this MonetaryValue